executor/src/main/resources/application.properties

```diff
@@ -6,3 +6,7 @@ mp.messaging.incoming.events-in.connector=smallrye-kafka
 mp.messaging.incoming.events-in.topic=events
 mp.messaging.incoming.events-in.value.serializer=org.apache.kafka.common.serialization.StringSerializer
 mp.messaging.incoming.events-in.auto.offset.reset=earliest
+
+# Configuring the outgoing channel (writing to Kafka) for Actions
+mp.messaging.outgoing.actions-out.connector=smallrye-kafka
+mp.messaging.outgoing.actions-out.value.serializer=org.apache.kafka.common.serialization.StringSerializer
```

Comment:
Fine for me, but does this impact your long term plan for Actions @robpblake ? We discussed it yesterday, but since I did not get it I'm wondering if this fits it

Reply:
@danielezonca  / @r00ta I've refactored to address this now as I think it helps to make my approach to Actions clearer. Please take a look

Reply:
Hi @robpblake , I think your refactoring is not following @danielezonca 's suggestion. 

He is suggesting to create **one** module **for each** action. We can have a generic module `actions` where we keep interfaces/api, but then every action is in a separated module `kafka-action`, `webhook-action` (eventually splitted between `kafka-action-api` and `kafka-action-core` for example), where we provide the beans that implement the factory interface.

Reply:
@r00ta Yeah I decided not to go to that level of decomposition just yet, keeping everything in one module for now. I'd like to see what comes out of https://issues.redhat.com/browse/MGDOBR-55 as to what other actions might  look like before we start to fully break it down like that. 

So this is the half-way option: Everything is centralised in a single location to avoid it spreading throughout the codebase e.g. not in `manager`, `shard`, `infra`, and if we want to further decompose in the future, we can.

Reply:
Yes I expect each action will have a different module but more in general ~all this code will be changed when we split `manager` because validation of the action can be probably moved in a `my-action-api` module in the `manager` while the execution semantic can stay closer to the `executor` in a `my-action-runtime` but for now I think it is enough to have a single `actions` module 🤔